spring-boot-istio-distributed-tracing-greeting-service/pom.xml

```diff
@@ -30,6 +30,15 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
```

Comment:
Missing `<version>${spring-boot.version}</version>`

Reply:
isn't this obtained from the `pluginManagement` tag on the `istio-distributed-tracing` project?

Reply:
sorry, it is.